nuxeo-retention-core/src/main/resources/OSGI-INF/retention-listener-contrib.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.retention.listeners">
+
+  <extension target="org.nuxeo.ecm.core.event.EventServiceComponent"
+    point="listener">
+    <listener name="retentionRecordCheckerListener"
+      class="org.nuxeo.ecm.retention.listener.RetentionRecordCheckerListener">
+      <event>checkRecordEvent</event> 
```

Comment:
Hmm, I think we should also have a way to trigger the archiving of a lists of docs ( as result of a query) and have a way not to double check for the documents already in this queue